staging/src/k8s.io/kubectl/pkg/cmd/apiresources/apiresources.go

```diff
@@ -205,18 +228,36 @@ func (o *APIResourceOptions) RunAPIResources() error {
 				APIGroupVersion: gv.String(),
 				APIResource:     resource,
 			})
+			apiResources = append(apiResources, resource)
+		}
+		if len(apiResources) > 0 {
```

Comment:
Don't filter out empty groups, this will cause that you won't print groups that don't have any resources. Filtering is happening earlier (there are `continue` blocks ensuring you'll skip group you don't care about).